kore/src/main/scala/org/kframework/unparser/Unparse.scala

```diff
@@ -34,9 +36,13 @@ object Unparse extends {
 object ToKast {
   def apply(k: K): String = {
     val b = StringBuilder.newBuilder
-    unparse(b, false, 0, k)
+    unparse(s => b ++= s, false, 0, k)
```

Comment:
it's not super readable I guess but that's a fault of overuse of operator overloading in Scala's standard library rather than anything in particular we've done. I'd prefer to leave this and then use java.lang.StringBuilder if this code gets reimplemented in Java which it probably will at some point.